nomad/jitsi-meet-backend.hcl

```diff
@@ -272,6 +272,46 @@ variable rtcstats_server {
   default = ""
 }
 
+variable jicofo_ssrc_rewriting {
+  type = string
+  default = "false"
+}
+
+variable jicofo_enable_opus_red {
+  type = string
+  default = "false"
+}
+
+variable jicofo_enable_h264 {
+  type = string
+  default = "true"
+}
+
+variable jicofo_enable_vp8 {
+  type = string
+  default = "true"
+}
+
+variable jicofo_enable_vp9 {
+  type = string
+  default = "true"
+}
+
+variable jicofo_strip_simulcast {
+  type = string
+  default = "true"
+}
+
+variable jicofo_visitors_max_participants {
+  type = string
+  default = "5"
+}
+
+variable jicofo_visitors_max_visitors_per_node {
+  type = string
+  default = "250"
+}
+
```

Comment:
These two by default are false ... I wonder what will be the value if those are false from ansible variables?

Reply:
And how that will be reflected in the docker config ...